minio/api.py

```diff
@@ -863,8 +863,12 @@ def copy_object(self, bucket_name, object_name, object_source,
         if sse:
             is_valid_sse_object(sse)
             headers.update(sse.marshal())
+        
+        # Uri-encode everything but the '/' character
+        copy_src = [ queryencode(part) for part in object_source.split('/') ]
+        copy_src = '/'.join(copy_src)
+        headers['X-Amz-Copy-Source'] = copy_src
```

Comment:
@johnnyvf24 fix is very simple 

```
queryencode(object_source).replace('%2F', '/')
```

Reply:
That is easier :+1:

Reply:
do you know why the checks are failing?

Reply:
https://github.com/minio/minio-py/pull/912/checks?check_run_id=646883805 can be checked right here

Reply:
I don't understand the error. It seems it is using a Makefile, but I don't see if it's a pylint failure, etc... Could you point me to a resource? Not sure why that one line would break the project.

Reply:
You should run `autopep8 --diff --exit-code *.py`